patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -458,14 +469,26 @@
              this.field_175730_i.removeAll(this.field_147483_b);
              this.field_147482_g.removeAll(this.field_147483_b);
              this.field_147483_b.clear();
++            
          }
++        
++        if (!this.tileEntitiesChunkToBeRemoved.isEmpty())
++        {
++    		for (Iterator<net.minecraft.util.math.ChunkPos> iterator2 = tileEntitiesChunkToBeRemoved.iterator(); iterator2.hasNext();) {
```

Comment:
1. braces
2. `for (ChunkPos pos : tileEntitiesChunkToBeRemoved)`
I believe World class already imported ChunkPos, is that true?